### ISSUE TITLE
Feature: Allow aria-hide multiple body childs.

### ISSUE THREAD

**Issue by Vanals** [reactjs/react-modal#842]
Hi, 

I saw that through `setAppElement` I can pass the id or directly the HTML reference of the element I want to aria hide.
In a CRA we usually have a #root element which contains all the HTML structure the user interact with.

But in other react  HTML structures you might not have a parent, such as #root, to simply hide. Instead, you might have multiple HTML elements directly child of the body.

Given that.. is there a way to aria-hide multiple elements and not just one?
Would be very useful! Or in alternative aria-hide everything a part from the modal container.

**Comment by diasbruno** [comment]
@Vanals For now, we can only handle one element.

One way to solve your problem would be to set `ariaHideApp={false}`, indicating that you will be in charge of setting the attribute to the correct elements (should help, for now)...and manage them using the method/way responsible for open the modal, and removing using the `onRequestClose`. 

https://github.com/reactjs/react-modal/blob/5594061cb7b75f27f9e1313bc5e517f32fb61039/src/helpers/ariaAppHider.js#L14

This function could receive a list of elements.

**Comment by Vanals** [comment]
> @Vanals For now, we can only handle one element.
> 
> One way to solve your problem would be to set `ariaHideApp={false}`, indicating that you will be in charge of setting the attribute to the correct elements (should help, for now)...and manage them using the method/way responsible for open the modal, and removing using the `onRequestClose`.
> 
> https://github.com/reactjs/react-modal/blob/5594061cb7b75f27f9e1313bc5e517f32fb61039/src/helpers/ariaAppHider.js#L14
> 
> This function could receive a list of elements.

Thank you Diasbruno!

Which one is the best way to import this function in my app?
Also, wouldn't the function `hide` do for my case? As what I need is to hide a certain number of element and `setElement()`, doesn't do that, right?

**Comment by diasbruno** [comment]
Maybe, you can try to use some of the react-modal internal stuff... 


```js
import * as ariaAppHider from "react-modal/helpers/ariaAppHider.js";
// ariaAppHider.hide and ariaAppHider.show

[DomElements].forEach(ariaAppHider.hide)
[DomElements].forEach(ariaAppHider.show)
```

The `setElement` is where it defines the element to use. If anyone has interest in submitting a PR for this, we could make this function to receive a list of elements.

**Comment by eemeli** [comment]
I'm pretty sure this was fixed by #861 in release 3.13.1.
